Assets/MixedRealityToolkit/_Core/Utilities/Editor/EditorProjectUtilities.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEditor;
+
+#if UNITY_EDITOR
```

Comment:
This can be removed since it's in an editor assembly

Reply:
done  :)